frontend/src/libs/components/icon/icons/icon-map.tsx

```diff
@@ -0,0 +1,16 @@
+import { type ValueOf } from '#/libs/types/types.js';
+import { ReactComponent as Arrow } from '#assets/icons/arrow.svg';
+import { ReactComponent as Plus } from '#assets/icons/plus.svg';
+import { ReactComponent as TrashBox } from '#assets/icons/trash-box.svg';
+import { IconNameToIcon } from '#libs/enums/enums.js';
+
+const iconNameToPlainSvgMap: Record<
+  ValueOf<typeof IconNameToIcon>,
+  React.FunctionComponent<React.SVGProps<SVGSVGElement>>
+> = {
+  [IconNameToIcon.ARROW]: Arrow,
```

Comment:
Please sync with Vlad about it

Reply:
[This](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/pull/75/commits/9ccea8dcdc0ab8060e3ab472e0f63f9843f03758) is how I implemented this, but mind Farid's [comment](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/pull/75#discussion_r1311837713)

Reply:
done